documentation/homework/week3/todo/src/App.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import './App.css';
+import moment from 'moment'
+
+// Components
+import AssignItem from './components/Assign'
+import ListItem from './components/ListItem'
+
+// Params
+import Params from './Params/locals'
+
+class App extends Component {
+  state = {
+    list: []
+  }
+  componentWillMount() {
+    this.setState({
+      list: Params.getLocals || []
+    })
+  }
+  Moments(date) {
+    const MOMENT_FORMAT = ["MM-DD-YYYY", "YYYY-MM-DD", "YYYY/MM/DD", "MM/DD/YYYY"]
+    return moment(date, MOMENT_FORMAT).format('LLLL')
+  }
+  handleRemove = (event, local_id) => {
+    this.setState({
+      list: Params.removeLocal(local_id)
+    })
+  }
+  handleAssign = (event) => { // Inserting Values
+    event.preventDefault()
+    const { deadline, description } = event.target
+    this.setState({
+      list: Params.saveLocal( // the Id is automaticly generats
+        Params.getLocals,
+        {
+          description: description.value,
```

Comment:
In your creation function here, it's important to reset the form values after we create a new todo, so that the form is reset and the inputs are empty. Right now the values stay inside the input, so if I want to add a second new todo, I have to delete my old values first.